accounts/abi/abi.go

```diff
@@ -274,12 +274,21 @@ func (abi ABI) Unpack(v interface{}, name string, output []byte) error {
 		// struct will match named return values to the struct's field
 		// names
 		case reflect.Struct:
+			idx := 0
```

Comment:
This cannot be what you use to reference the toGoType, as its needed to move through the bytecode.